FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,47 @@
+def fizzBuzz(isFB)
+
+  def printFizzBuzz(isFB)
+    if
+      isFB%3 == 0 && isFB%5 == 0
+      puts "FizzBuzz"
+    elsif isFB%3 == 0
+      puts "Fizz"
+    elsif isFB%5 == 0
+      puts "Buzz"
+    else
+      puts "No FizzBuzz for you, sport!"
+    end
+  end
+
+  def isNumber(isNum)
+    isNum = isNum.to_s
+    /\A[+-]?\d+(\.[\d]+)?\z/.match isNum
```

Comment:
While I certainly appreciate the art of a good regexp, I think the Ruby itself provides nice string to float parsing. Might want to take a look at https://ruby-doc.org/core-2.6.3/Kernel.html#method-i-Float . Please keep in mind that `exception: false` is a new addition to the language(I think from version 2.6).

If you're curious what's happening underhood when you call `Float('123.123')` there's a pretty cute regexp there as well ;) https://stackoverflow.com/questions/1034418/determine-if-a-string-is-a-valid-float-value

Reply:
Yep, I'm aware that even though my regexp works - it was not an elegant solution. Thanks for pointing me in that direction - I'll read into it and definitely this method will be used.